section4/exercises/dog.rb

```diff
@@ -19,6 +19,13 @@ def bark
   def eat
     @hungry = false
   end
+  
+  def play
+    @hungry = true
```

Comment:
Nice—here we're changing the `@hungry` instance variable (attribute). 

Good example to compare to methods in the Burrito exercise that aren't altering instance variables